src/components/button/button.spec.tsx

```diff
@@ -102,6 +133,57 @@ describe("Button", () => {
     });
   });
 
+  describe( "Button Minor Padding",() => {
+      it.each(isMinorSizesPadding)("when size is %s the padding is %s", (size, padding) => {
+          const wrapper = mount(
+            <Button isMinor size={size}>Foo</Button>
+          )
+          assertStyleMatch({padding}, wrapper)    
+      });
+  });
+
+  describe( "Button Minor Colors ",() => {
+    it.each(isMinorColors)("when buttonType is %s", (buttonType, styles) => {
+      const wrapper = mount(
+        <Button isMinor buttonType={buttonType} >Foo</Button>
+      )
+      assertStyleMatch({...styles}, wrapper)    
+  });
+
+  describe( "Button Minor Colors for destructive and disabled",() => {
+    it('matches the style for the Button Minor destructive', () => {
```

Comment:
This isn't testing is `destructive` is true (see above for proposed changes above)